keras/layers/normalization.py

```diff
@@ -35,6 +35,7 @@ class BatchNormalization(Layer):
         weights: Initialization weights.
             List of 2 Numpy arrays, with shapes:
             `[(input_shape,), (input_shape,)]`
+			Note that the order of this list is [Gamma, Beta, mean, std]
```

Comment:
This is getting the PEP8 test to fail. You don't need this crazy indentation. You don't need to capitalize the variable names.